### PR TITLE
(BIDS-2019) move Time in a template

### PR DESCRIPTION
**Pull request by KostasGiannakakis**
### <samp>🤖 Generated by Copilot at 05f719b</samp>

Added a reusable `timeToggle` component to show and switch the time format of an epoch. Used the component in the `epoch.html` template to replace the original code.

### CODE REVIEW

File: templates/components/timeToggle.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{ define "timeToggle" }}
+  <div class="row border-bottom p-3 mx-0">
+    <div class="col-md-2">Time:</div>
```
Comment (D13ce) [inline templates/components/timeToggle.html:3]: This breaks the formatting:
image
The time now is placed to much to the left.

File: templates/components/timeToggle.html
Comment (D13ce) [inline templates/components/timeToggle.html:1]: While I really like the idea to put this into a separate file ... I am REALLY confused by the fact that you did not make any use of that refactoring. There are many places where you could use it. Simply search, for example, for `setTs()` and you find about 4 more locations.

**Review by D13ce — CHANGES_REQUESTED** [review]
Left some comments, ptal

File: templates/components/timeToggle.html
Comment (KostasGiannakakis) [reply templates/components/timeToggle.html:1]: I am already aware of that. I wanted to first send this one, and make sure it works properly, before proceeding with the rest of the refactoring. I should have explained it with a comment under this PR.

File: templates/components/timeToggle.html (hunk 1 above)
Comment (KostasGiannakakis) [reply templates/components/timeToggle.html:3]: will take a look for this one. thnx

**Review by D13ce — CHANGES_REQUESTED** [review]
Not all comments have been solved yet.

File: templates/components/timeToggle.html
Comment (KostasGiannakakis) [reply templates/components/timeToggle.html:1]: @D13ce I updated all relevant files, to include the new template for the timeRow

**Review by D13ce — CHANGES_REQUESTED** [review]
Not a single page out of the 4 ones you changed with your latest commits work. Please check.

File: templates/slot/slotFuture.html
Comment (D13ce) [inline templates/slot/slotFuture.html:1]: This change breaks the position of the time value, it is not aligned properly anymore (compare it with the slot line above):

image

**Review by D13ce — CHANGES_REQUESTED** [review]
I have found two placement issues, see my comments below.

At this point I believe that at most only the 3 buttons should be part of `timeRow.html` and the file should be renamed accordingly. Thoughts?

**Comment by KostasGiannakakis** [comment]
I decided to revert back to an easier approach. The template for the timeRow I created it seems to have some inconsistencies and it led to require more work than I was originally expecting. With the `autocomplete="off"` approach, the issue is being solved. I will look again in the future whether it is possible to create more reusable components for our frontend. 
cc. @D13ce

**Review by D13ce — APPROVED** [review]
lgtm

**Pull request merged**